### PR TITLE
Add e2e tests

### PR DESCRIPTION
**Pull request by benjick**
Solves #18

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,28 @@
 language: node_js
 node_js:
   - "8"
-  - "9"
```
Comment (zimme) [inline .travis.yml]: We should probably test it on version 10 also

Comment (zimme) [reply .travis.yml]: But I guess this doesn't really matter as it's testing inside a docker container...

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
+before_script:
+  - sleep 3
+  - docker-compose up -d --build
+
 script:
   - yarn lint
+  - yarn ci
+
+after_script:
+  - docker-compose down
```
Comment (zimme) [inline .travis.yml]: We might need to add
```
services:
  - docker
```
to this file?

Comment (benjick) [reply .travis.yml]: I'm not really sure, it runs the test 👍

Comment (zimme) [reply .travis.yml]: Yeah, maybe adding this just gives you access to the docker command and as we don't use that it doesn't really matter

File: cypress/.eslintrc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "env": {
+    "jest": true
+  },
+  "globals": {
+    "cy": true,
+    "context": true
+  }
+}
```
Comment (zimme) [inline cypress/.eslintrc]: This should be named .eslintrc.json, using eslintrc file without extension is deprecated.

**Pull request merged**